Listing '.'...
Listing './bin'...
Listing './include'...
Listing './lib'...
Listing './lib/python3.5'...
Listing './lib/python3.5/site-packages'...
Listing './lib/python3.5/site-packages/argparse-1.4.0.dist-info'...
Listing './lib/python3.5/site-packages/colorama'...
Listing './lib/python3.5/site-packages/colorama-0.3.9.dist-info'...
Listing './lib/python3.5/site-packages/dotcloud'...
Listing './lib/python3.5/site-packages/dotcloud/client'...
Listing './lib/python3.5/site-packages/dotcloud/packages'...
Listing './lib/python3.5/site-packages/dotcloud/ui'...
Compiling './lib/python3.5/site-packages/dotcloud/ui/cli.py'...
***   File "./lib/python3.5/site-packages/dotcloud/ui/cli.py", line 135
    except RESTAPIError, e:
                       ^
SyntaxError: invalid syntax

Compiling './lib/python3.5/site-packages/dotcloud/ui/config.py'...
***   File "./lib/python3.5/site-packages/dotcloud/ui/config.py", line 32
    os.mkdir(self.dir, 0700)
                          ^
SyntaxError: invalid token

Compiling './lib/python3.5/site-packages/dotcloud/ui/utils.py'...
***   File "./lib/python3.5/site-packages/dotcloud/ui/utils.py", line 22
    print ''
           ^
SyntaxError: Missing parentheses in call to 'print'

Listing './lib/python3.5/site-packages/dotcloud-0.9.8.egg-info'...
Listing './lib/python3.5/site-packages/pip'...
Listing './lib/python3.5/site-packages/pip/_vendor'...
Listing './lib/python3.5/site-packages/pip/commands'...
Listing './lib/python3.5/site-packages/pip/compat'...
Listing './lib/python3.5/site-packages/pip/models'...
Listing './lib/python3.5/site-packages/pip/operations'...
Listing './lib/python3.5/site-packages/pip/req'...
Listing './lib/python3.5/site-packages/pip/utils'...
Listing './lib/python3.5/site-packages/pip/vcs'...
Listing './lib/python3.5/site-packages/pip-9.0.1.dist-info'...
Listing './lib/python3.5/site-packages/pkg_resources'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor'...
Listing './lib/python3.5/site-packages/pkg_resources/_vendor/packaging'...
Listing './lib/python3.5/site-packages/pkg_resources/extern'...
Listing './lib/python3.5/site-packages/pkg_resources-0.0.0.dist-info'...
Listing './lib/python3.5/site-packages/requests'...
Listing './lib/python3.5/site-packages/requests/packages'...
Listing './lib/python3.5/site-packages/requests/packages/charade'...
Listing './lib/python3.5/site-packages/requests/packages/urllib3'...
Listing './lib/python3.5/site-packages/requests/packages/urllib3/contrib'...
Compiling './lib/python3.5/site-packages/requests/packages/urllib3/contrib/ntlmpool.py'...
***   File "./lib/python3.5/site-packages/requests/packages/urllib3/contrib/ntlmpool.py", line 38
    """
     ^
SyntaxError: (unicode error) 'unicodeescape' codec can't decode bytes in position 130-131: truncated \uXXXX escape

Listing './lib/python3.5/site-packages/requests/packages/urllib3/packages'...
Listing './lib/python3.5/site-packages/requests/packages/urllib3/packages/ssl_match_hostname'...
Listing './lib/python3.5/site-packages/requests-1.2.3.egg-info'...
Listing './lib/python3.5/site-packages/setuptools'...
Listing './lib/python3.5/site-packages/setuptools/command'...
Listing './lib/python3.5/site-packages/setuptools/extern'...
Listing './lib/python3.5/site-packages/setuptools-32.3.1.dist-info'...
Listing './share'...
Listing './share/python-wheels'...
